docs/_static/usecase/v1.yaml

```diff
@@ -0,0 +1,190 @@
+                    items:
+                      type: string
+                      description: ID of the presentation that was removed from the list.
+        default:
+          $ref: "../common/error_response.yaml"
+    post:
+      summary: Adds a presentation to the list
+      description: |
+        An API provided by the use case maintainer that adds a presentation to the list.
+        The presentation must be signed by subject of the credentials it contains.
+        
+        error returns:
+        * 400 - incorrect input; e.g. unsupported presentation or credential type, invalid signature, unresolvable credential subject, etc.
+      operationId: addPresentation
+      tags:
+        - usecase
+      requestBody:
+        description: The presentation to add to the list
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/VerifiablePresentation"
+      responses:
+        "201":
+          description: Presentation was added to the list
+        "400":
+          $ref: "../common/error_response.yaml"
+        default:
+          $ref: "../common/error_response.yaml"
+    delete:
+      summary: Delete a credential subject from list.
+      description: |
+        An API provided by the use case maintainer that removes all presentations of a specific credential subject from the list.
+        The presentation must be signed by credential subject which' presentations should be removed.
+        The presentation should not contain any credentials but if there are, they are ignored.
+
+        error returns:
+        * 400 - incorrect input; e.g. unsupported presentation, invalid signature, etc.
+      operationId: removePresentations
+      tags:
+        - usecase
+      requestBody:
+        description: Presentation identifying the credential subject to remove from the list.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/VerifiablePresentation"
+      responses:
+        "201":
+          description: Presentation was added to the list
+        "400":
+          $ref: "../common/error_response.yaml"
+        default:
+          $ref: "../common/error_response.yaml"
+  /usecase/{listName}/search:
+    parameters:
+      - name: listName
+        in: path
+        required: true
+        schema:
+          type: string
+      # Way to specify dynamic query parameters
+      # See https://stackoverflow.com/questions/49582559/how-to-document-dynamic-query-parameter-names-in-openapi-swagger
+      - in: query
+        name: query
+        schema:
+          type: object
+          additionalProperties:
+            type: string
+        style: form
+        explode: true
+    get:
+      summary: Searches a for presentations on the usecase list.
+      description: |
+        An API of the use case client that searches a for presentations on the usecase list,
```

Comment:
it's a bit weird to have the client and server api on the same path. Before a client can use it as registry some additional work has to be done (did service resolving). I think the result of the list should feed into the same registry as the network for now. An api can be added later.

Reply:
Then we have to build pruning into that store (it's the "VCR store") as well. Maybe we should just search in both stores. Although the current VCR search does not know about the discovery service IDs, so you don't know in which service to search.

Perhaps search in all of them, and deduplicate results?